components/script/dom/console.rs

```diff
@@ -93,18 +93,30 @@ impl Console {
                 format!("{}: {}ms", label, delta)
             );
             println!("{}", message);
-            Self::send_to_devtools(global, LogLevel::Log, message);
+            Self::send_to_devtools(global, LogLevel::Log, message, DOMString::new());
         };
     }
+
+    // https://developer.mozilla.org/en-US/docs/Web/API/Console/group
+    pub fn Group(global: &GlobalScope, label: Option<DOMString>) {
+        let label = label.unwrap_or_else(|| DOMString::from("<no group label>"));
+        Self::send_to_devtools(global, LogLevel::Group, label.clone(), label);
+    }
+
+    // https://developer.mozilla.org/en-US/docs/Web/API/Console/groupEnd
+    pub fn GroupEnd(global: &GlobalScope) {
+        Self::send_to_devtools(global, LogLevel::GroupEnd, DOMString::new(), DOMString::new());
+    }
 }
 
-fn prepare_message(log_level: LogLevel, message: DOMString) -> ConsoleMessage {
+fn prepare_message(log_level: LogLevel, message: DOMString, group_name: DOMString) -> ConsoleMessage {
```

Comment:
Are you suggesting removing the argument from prepare_message and, since we are sending the label as the message, using that instead of the extra argument?